tcms/core/tests/test_views.py

```diff
@@ -82,6 +93,22 @@ def test_check_base_url_configured(self):
             response = self.client.get("/", follow=True)
             self.assertNotContains(response, self.base_url_error_message)
 
+    @unittest.skipUnless(
+        os.getenv("TEST_DASHBOARD_CHECK_SSL"),
+        "Checking SSL is not enabled",
+    )
+    def test_check_connection_uses_ssl(self):
+        response = self.client.get("/", follow=True)
+        self.assertNotContains(response, self.ssl_error_message)
+
+    @unittest.skipUnless(
+        not os.getenv("TEST_DASHBOARD_CHECK_SSL"),
+        "Check not using SSL is not enabled",
+    )
```

Comment:
Remove the conditional decorator. This test case should be executed unconditionally.